src/apps/dashboard/controllers/networking.js

```diff
@@ -142,6 +146,14 @@ export default function (view) {
         page.querySelector('#chkEnableIP6').checked = config.EnableIPv6;
         page.querySelector('#chkEnableIP4').checked = config.EnableIPv4;
         page.querySelector('#txtPublishedServer').value = (config.PublishedServerUriBySubnet || []).join(', ');
+        const selectPublicUserListing = page.querySelector('#selectPublicUserListing');
```

Comment:
I would separate the assignment with blank lines (as I did in my previous suggestion).
```suggestion

        const selectPublicUserListing = page.querySelector('#selectPublicUserListing');

```